docs/components_and_bundles/components/Locale/basic_usage.rst

```diff
@@ -4,7 +4,16 @@ Basic Usage
 LocaleContext
 -------------
 
-...
+In the Locale component there are three LocaleContexts defined:
+* ``CompositeLocaleContext``
+* ``ImmutableLocaleContext``
+* ``ProviderBasedLocaleContext``
+
+CompositeLocaleContext
+~~~~~~~~~~~~~~~~~~~~~~
+
+	It is a composite of different contexts available in your application, which are prioritized while being injected here (the one with highest priority is used).
+	It has the ``getLocaleCode``, ``_construct()``, and ``addContext(LocaleContextInterface $localeContext, int $priority)`` methods available.
```

Comment:
This line is quite fragile and can be replaced by looking at the source code instead.